manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
+PackageLocale: en-US
+Publisher: Ellie Mae, Inc.
+PackageName: Encompass360
+License: Proprietary
+Copyright: Copyright (C) 2024 ICE Mortgage Technology, Inc.
+ShortDescription: Encompass360 Installer

```

Comment:
Please provide a better description of what Encompass is, does, or what it is used for